client/lib/index.js

```diff
@@ -352,6 +367,10 @@ module.exports = class Client extends PassThrough {
 					}
 				} catch (e) {
 					capturedError = e;
+					// If you catch an here, then uncomment the following to debug:
```

Comment:
does this need to be optional? Could we just print the error if we catch it every time?

Reply:
It ain't optional. We are printing the error, check line 371. I will improve my statement here. What I meant was, that people can uncomment the throw statement if they want to get a stack of errors and see which line is actually causing it. 
 
It works well, the test stops at the right moment and we get the visibility over the client error without the client breaking down in a bad way. Win win.